interfaces/PA-App/src/app/shared/play-text-audio/play-text-audio.component.ts

```diff
@@ -112,6 +112,7 @@ export class PlayTextAudioComponent implements OnInit {
     this.state = state;
     this.setLabel(state);
     this.setIcon(state);
+    this.isDisabled = false;
```

Comment:
```suggestion
```
I think a better place for it would be in the `createPlayer()`-method.
So, here: https://github.com/global-121/121-platform/pull/675/files#diff-eedc6515a85de1b46dc16ef779464ef6R70-R79
```ts
  private createPlayer() {
    this.isDisabled = false;
    this.player = new Howl({
// ...
    });
```
So you always start 'with a clean slate'.
Resetting the `isDisabled`-property _every_ time its state is set feels a bit too much.